systemtest/src/test/java/io/strimzi/systemtest/cruisecontrol/CruiseControlST.java

```diff
@@ -250,4 +253,43 @@ void testCruiseControlReplicaMovementStrategy(ExtensionContext extensionContext)
         ccConfFileContent = cmdKubeClient(namespaceName).execInPodContainer(ccPodName, Constants.CRUISE_CONTROL_CONTAINER_NAME, "cat", Constants.CRUISE_CONTROL_CONFIGURATION_FILE_PATH).out();
         assertThat(ccConfFileContent, containsString(newReplicaMovementStrategies));
     }
+
+    @ParallelNamespaceTest
+    void testCruiseControlIntraBrokerBalancing(ExtensionContext extensionContext) {
+        final String namespaceName = StUtils.getNamespaceBasedOnRbac(namespace, extensionContext);
+        String clusterName = mapWithClusterNames.get(extensionContext.getDisplayName());
```

Comment:
Could you use TestStorage class? You can look at `testCruiseControlBasicAPIRequests` test case.
```java
final TestStorage testStorage = new TestStorage(extensionContext);
```

Reply:
@see-quick just made the changes, let me now what you think!